api/index.js

```diff
@@ -0,0 +1,45 @@
+const router = require('express').Router();
+const { User, Relationship, Family } = require('../db/models');
+const Sequelize = require('sequelize');
+const Op = Sequelize.Op;
+
+//get all users
+router.get('/users', (req, res, next) => {
+    User.findAll()
+        .then(users => res.send(users))
+        .catch(next)
+});
+
+//get user by id
+router.get('/users/:id', (req, res, next) => {
+    User.findByPk(req.params.id)
+        .then(user => res.send(user))
+        .catch(next)
+});
+
+//get users relationships
+router.get('/users/:id/relationships', (req, res, next) => {
```

Comment:
You probably only want to return this users relationships. This route would return their relationships, and other users relationships to them. It would probably be simpler to just use getRelationships() or just search for { userId: req.params.id }

Reply:
Hey just fixed this!